ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -1881,6 +1881,22 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4
+- as: baremetal-upi-ovn-ipv4-p1-f14
+  cluster: build05
+  cron: 30 21 7,21 * *
+  steps:
+    cluster_profile: equinix-ocp-metal-qe
+    env:
+      AUX_HOST: openshift-qe-bastion.arm.eng.rdu2.redhat.com
```

Comment:
Expose internal info to outside of RH will trigger security alerts

Reply:
already discussed and solved.

Reply:
jut fyi, the domain is exposed since a while now and the IT should have disabled those alerts.

Reply:
https://github.com/openshift/release/blob/master/clusters/build-clusters/build05/assets/dns.yaml

Reply:
Good to know. Thanks for the info.

Is it possible to set a default value for AUX_HOST, so that we do not need to specify AUX_HOST in each test ?

Reply:
Not now. Maybe later, but for now I prefer keeping the AUX_HOST value separate from the implemented steps for safety reasons. When the efforts regarding joining the infra resources will be concluded, the meaning and scope of AUX_HOST will be rediscussed.